demo/client.cc

```diff
@@ -19,10 +22,13 @@ bool user_specific = false;
 // Paramters used to build the request.
 content_analysis::sdk::AnalysisConnector connector =
     content_analysis::sdk::FILE_ATTACHED;
-std::string request_token = "req-12345";
+time_t request_token_number = time(nullptr);
```

Comment:
I'm not sure what assigning `nullptr` to a `time_t` does. Shouldn't this be a regular int?

Reply:
The arg is an alternate way to get the return value.

https://man7.org/linux/man-pages/man2/time.2.html